cmake/config/template-arguments.in

```diff
@@ -55,42 +54,42 @@ SERIAL_VECTORS := { Vector<double>;
 
 // same as SERIAL_VECTORS but only with real-valued PETSc vectors
 REAL_SERIAL_VECTORS := { Vector<double>;
-                    Vector<float> ;
+                         Vector<float> ;
```

Comment:
I like the reindentation, but would you mind putting indentation fixes in another commit? It makes it harder to read which lines are removed.